spidermon/contrib/scrapy/monitors/monitors.py

```diff
@@ -459,8 +459,8 @@ def test_check_if_field_coverage_rules_are_met(self):
             "SPIDERMON_FIELD_COVERAGE_RULES"
         )
         for field, expected_coverage in field_coverage_rules.items():
-            actual_coverage = self.data.stats.get(
-                f"spidermon_field_coverage/{field}", 0
+            actual_coverage = round(
```

Comment:
Alternatively, to avoid rounding  we can change the comparation with:

```python
if  actual_coverage > expected_coverage or math.isclose(actual_coverage, expected_coverage, abs_tol=threshold)  :
   continue
else:
    failure.append(
```
    
 In any case, the margin of error should be configurable, and the default configuration should be do nothing as compared to the current implementation. Otherwise, this would break backward compatibility and should be clearly warned